SemanticUI/Modules/Common.elm

```diff
@@ -0,0 +1,9 @@
+module SemanticUI.Modules.Common exposing (HtmlBuilder)
```

Comment:
I am not a fan of `Common` modules. It doesn't really have anything to do with `SemanticUI`, so I'd take it out of that namespace. How about just `Html.HtmlBuilder`?

Reply:
Good idea, I like it.

Reply:
Ah I missed the part about `Html.HtmlBuilder` namespace. Will do that